tests/testthat/test-util.R

```diff
@@ -1,4 +1,14 @@
 test_that("cache key", {
   expect_identical(bm_run_cache_key("foo", alpha = "one", beta = 2), "foo/one-2")
   expect_identical(bm_run_cache_key("foo", beta = 2, alpha = "one"), "foo/one-2")
+})
+
+test_that("find_r()", {
+  out <- system(paste(find_r(), "--no-save -s 2>&1"), intern = TRUE, input = "print('output')\n")
+  expect_match(out, "output")
+
+  # the stderror isn't redirected correctly on windows, at least in GHA
+  skip_on_os("windows")
```

Comment:
Rather than skip on windows, I'd just comment windows out of the GHA build matrix and leave a note there (and make an Issue) that we will need to do work to support windows. Sounds like it fundamentally doesn't work on windows, so there's no point in papering over that by skipping tests.